src/types.rs

```diff
@@ -1,15 +1,29 @@
 //! generic types for GPX
 
 use geo::{Geometry, ToGeo};
-use geo::{LineString, MultiLineString, Point};
+use geo::{Bbox, LineString, MultiLineString, Point};
 
 use chrono::DateTime;
 use chrono::prelude::Utc;
 
+#[derive(Clone, Copy, Debug, PartialEq)]
+pub enum GpxVersion {
+    Unknown,
+    Gpx10,
```

Comment:
Any thoughts on alternative naming, like `GpxVersion::OneZero`, `GpxVersion:Gpx1_0`? Also, from above, the `Unknown` value is questionable.

Reply:
The Unknown value is needed for consume when we don't know the version yet.

Reply:
`GpxVersion::Gpx1_0` trigger a warning from the compiler. I don't find `GpxVersion::OneZero` intuitive.